man/systemd.exec.xml

```diff
@@ -366,6 +366,22 @@
         <xi:include href="version-info.xml" xpointer="v233"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>BindJournalSockets=</varname></term>
+
+        <listitem><para>Takes a boolean argument. If true, sockets from <citerefentry>
+        <refentrytitle>systemd-journald.socket</refentrytitle><manvolnum>8</manvolnum></citerefentry>
+        will be bind mounted into the mount namespace. This is particularly useful when a different rootfs
+        is used (e.g. through <varname>RootDirectory=</varname> or <varname>RootImage=</varname>), as
+        <varname>MountAPIVFS=yes</varname> mounts a fresh <filename>/run/</filename> inside the new root.</para>
```

Comment:
Hm, with this patch, `MountAPIVFS=yes` implies this, so `/run/` is not 'fresh' unless the new knob is disabled...

The sentence sounds that explicitly setting `BindJournalSockets=yes` is suggested when `RootDirectory=` or friends is specified. But, that's not true. Confusing.

Reply:
It is "fresh", i.e. detached from the `/run/` from host. We do create e.g. propagation dirs in there, yet those including the journal sockets are auxiliary stuff, which doesn't change the fact that a different tmpfs is in use.

> The sentence sounds that explicitly setting BindJournalSockets=yes is suggested when RootDirectory= or friends is specified. But, that's not true. 

Hmm? It doesn't hurt to enable it manually? (We even do that ourselves in portable profiles)

Reply:
It does not hurt. But it is not necessary, as it is implied, as explained in the next paragraph. So, I confused.

Reply:
I got rid of the explicit mentioning of "usage of different rootfs" now. That should prevent the confusion?